sites/partners/src/components/listings/PaperListingForm/UnitsSummaryAmiForm.tsx

```diff
@@ -74,6 +74,12 @@ const UnitsSummaryAmiForm = ({
     [amiInfo, setAmiPercentageOptions]
   )
 
+  useEffect(() => {
```

Comment:
There is an edge case only appearing on the cypress tests where the amiChartId is not being updated before the onChange event happens. By moving it to the useEffect we can guarantee that the order will happen correctly